README.md

```diff
@@ -0,0 +1,35 @@
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
+# Synapse Operator
+
+A Juju charm deploying and managing Synapse on Kubernetes. Synapse is a drop in
+replacement for other chat servers like Mattermost and Slack.
+
+This charm simplifies initial deployment and "day N" operations of Synapse
+on Kubernetes, such as integration with SSO, access to S3 for redundant file
+storage and more. It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
+offerings.
+
+As such, the charm makes it easy for those looking to take control of their own
+Chat server whilst keeping operations simple, and gives them the
+freedom to deploy on the Kubernetes platform of their choice.
+
+For DevOps or SRE teams this charm will make operating Synapse simple and
+straightforward through Juju's clean interface. It will allow easy deployment
+into multiple environments for testing of changes.
+
+## Project and community
+
+The Synapse Operator is a member of the Ubuntu family. It's an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+* [Contribute](https://charmhub.io/indico/docs/contributing)
```

Comment:
There was some spec from @mthaddon regarding an update regarding community contributions, should we incorporate that or wait for a bit?

Reply:
Since this is just an initial commit to prepare the repository and there are stories regarding code/docs I think we could address that later.